common/beerocks/bwl/common/key_value_parser.cpp

```diff
@@ -0,0 +1,225 @@
+
+#include <bwl/key_value_parser.h>
+
+#include <bcl/beerocks_state_machine.h>
+#include <bcl/beerocks_string_utils.h>
+#include <bcl/network/network_utils.h>
+
+#include <easylogging++.h>
+#include <sstream>
+
+using namespace bwl;
+
+namespace bwl {
+constexpr char EVENT_KEYLESS_PARAM_OPCODE[] = "_opcode";
```

Comment:
Coding style:
```suggestion
constexpr char event_keyless_param_opcode[] = "_opcode";
```

However:
- probably this should be a std::string instead of char[];
- I would make this stuff parameters of the constructor rather than hardcoding.

Reply:
- It is not possible to define `std::string` as `constexpr`.
- I used capital letters since constexpr is like a Macro.